### PR TITLE
Disambiguate trackers vs. trackers blocked vs. total tracker requests on page

### PR DESCRIPTION
**Pull request by laurengarcia**
**Reviewer:**
@russellholt @jdorweiler 

## Description:
There has been some miscommunication around what the tracker UX was supposed to display. The page-level trackerlist sliding subview was previously only displaying the num of requests (listed by company) that we *actually blocked*. What it should have been showing was all the requests (listed by company) on the page, *whether we blocked those or not*. Part of the reason for this miscommunication I think was the mislabeling/need for disambiguation in the Tab class around trackers on page (now labeled simply `trackers` vs. previously was `potentialBlocked`, and now `trackersBlocked` is the new name for `trackers` -- much more clear and usable for future developers who would probably have been very confused). So that's it now -- we have `tab.trackers` and `tab.trackersBlocked`. Previously, what is now `tab.trackers` (was `tab.potentialBlocked`) was not wired up in the same manner as `tab.trackersBlocked`, so those two objects are identical now (previously it was only an array of urls). I also changed some of the labels on the UI. See attached image.

One issue: when user whitelists a site, the text "0/6 unique trackers blocked" overlaps with the toggle button in the main popup view.

Tests probably need fixing. Opening PR now so we can discuss UX.

disambiguated-extension2

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

**Comment by jdorweiler** [comment]
👍 for name changes. Even I was getting confused.

**Comment by laurengarcia** [comment]
first rule of programming: naming things is hard.

File: js/templates/trackerlist-truncated.es6.js [hunk 1]
```diff
@@ -6,13 +6,13 @@ module.exports = function () {
...
         return bel`<section class="top-blocked card">
-            <h3 class="menu-title">Top blocked over time</h3>
+            <h3 class="menu-title">Top blocked companies over time</h3>
             <ul class="menu-list top-blocked__list">
                 ${trackerListItems(this.model.companyListMap)}
                 <li class="top-blocked__li top-blocked__li--see-all border--top">
                     <a href="#" class="link-secondary js-top-blocked-see-all">
                         <span class="icon icon__arrow pull-right"></span>
-                        See all
+                        See tracker request counts
```
Comment (jdorweiler) [inline js/templates/trackerlist-truncated.es6.js]: Seems weird to me to have "See tracker request counts" twice on the popup but going to separate views.

Comment (laurengarcia) [inline js/templates/trackerlist-truncated.es6.js]: Yeah, I changed these labels a couple times. Suggestions?

Comment (russellholt) [inline js/templates/trackerlist-truncated.es6.js]: Tracker request details?
Page details?

**Comment by laurengarcia** [comment]
Updated with UI changes we discussed yesterday.

Unit tests are not updated here (yet). I can send test updates in another PR for those later today if we want to merge this now to keep our place in line in FF store, that's fine by me.

**Comment by jdorweiler** [comment]
LGTM

**Comment by laurengarcia** [comment]
Ok, can revert the change we discussed. The toggle will go back, but the label can't say "Unique tracker requests" because that's not what it is. It is a count of the "unique trackers blocked", not requests. I'm assuming you meant you want to revert it to "unique trackers blocked", but then we still have the horizontal space issue if we put the toggle back.

"Tracker request details" for the see-all link sounds fine to me.

**Comment by russellholt** [comment]
@laurengarcia yes you're right about "unique trackers blocked". Not requests.

I feel like we will eventually come up with a much better way to communicate the idea that, when whitelisted, or for whatever reason, there are n trackers on this page but we're blocking m, or  we're not actually blocking them because it's off.. in a super concise way. The "0/6" method feels like an easy short term compromise, not really a long-term solution

Maybe just

normal: 6 unique trackers blocked (green background)
whitelisted: 6 unique trackers (red background)

not sure. I know we're not supposed to differentiate only by color though, for accessibility.

**Comment by laurengarcia** [comment]
Ok, updated per discussion. Final screenshot:

final

File: js/models/site.es6.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
-        'none':     'Secure Connection Unavailable'
-    };
+    'default':  'Secure Connection',
+    'upgraded': 'Forced Secure Connection',
+    'none':     'Secure Connection Unavailable'
+};
+
+const whitelistStates = {
+    'isWhitelisted': 'Tracker blocking disabled',
+    'notWhitelisted': 'Tracker blocking enabled',
```
Comment (russellholt) [inline js/models/site.es6.js]: 👍

File: js/models/site.es6.js [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
-        'none':     'Secure Connection Unavailable'
-    };
+    'default':  'Secure Connection',
+    'upgraded': 'Forced Secure Connection',
+    'none':     'Secure Connection Unavailable'
+};
+
+const whitelistStates = {
+    'isWhitelisted': 'Blocking off (this domain)',
+    'notWhitelisted': 'Blocking on (this domain)',
```
Comment (russellholt) [inline js/models/site.es6.js]: 👍

File: js/models/site.es6.js [hunk 4]
```diff
@@ -50,8 +56,8 @@ Site.prototype = $.extend({},
 
       updateTrackerCount: function() {
           if (this.tab) {
-            this.trackerCount = this.tab.getBadgeTotal();
-            this.potential = Object.keys(this.tab.potentialBlocked).length;
+            this.trackersCount = this.tab.getUniqueTrackersCount();
```
Comment (russellholt) [inline js/models/site.es6.js]: this is better

**Pull request merged**